docs/src/content/en/docs/sidebars.js

```diff
@@ -397,89 +397,51 @@ const sidebars = {
         {
           type: "doc",
           id: "deployment/building-mastra",
-          label: "Building Mastra",
+          label: "Mastra Server",
         },
         {
           type: "doc",
           id: "deployment/monorepo",
-          label: "With a Monorepo",
+          label: "Monorepo",
+        },
+        {
+          type: "doc",
+          id: "deployment/cloud-providers",
+          label: "Cloud Providers",
         },
         {
           type: "doc",
           id: "deployment/web-framework",
-          label: "With a Web Framework",
+          label: "Web Framework",
         },
+      ],
+    },
+    {
+      type: "category",
+      label: "Mastra Cloud",
+      items: [
         {
-          type: "category",
-          label: "Mastra Cloud",
-          items: [
-            {
-              type: "doc",
-              id: "deployment/mastra-cloud/overview",
-              label: "Overview",
-            },
-            {
-              type: "doc",
-              id: "deployment/mastra-cloud/setting-up",
-              label: "Setup & Deploy",
-            },
-            {
-              type: "doc",
-              id: "deployment/mastra-cloud/dashboard",
-              label: "Dashboard",
-            },
-            {
-              type: "doc",
-              id: "deployment/mastra-cloud/observability",
-              label: "Observability",
-            },
-          ],
+          type: "doc",
+          id: "mastra-cloud/overview",
+          label: "Overview",
+          customProps: {
+            tags: ["beta"],
```

Comment:
If think the beta tag should be on the "Mastra Cloud" top-level item. Does that work?